### PR TITLE
feat: JOIN-27736 custom theme

### PR DESCRIPTION
**Pull request by MichalBaran94**
[JOIN-27736](https://joinsolutionsag.atlassian.net/browse/JOIN-27736)

[JOIN-27736]: https://joinsolutionsag.atlassian.net/browse/JOIN-27736?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/docs/joinTheme.copy.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      <meta name="msapplication-TileColor" content="#ff0000" />
+      <meta name="theme-color" content="#ffffff" />
+    </>
+  ),
+  search: true,
+  prevLinks: true,
+  nextLinks: true,
+  footer: true,
+  footerEditLink: "Edit this page on GitHub",
+  footerText: <>© {new Date().getFullYear()} Cal.com, Inc. All rights reserved.</>,
```
Comment (dmkav) [inline apps/docs/joinTheme.copy.js:44]: cal.com? :)

File: apps/docs/joinTheme.copy.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        name="og:description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:image" content={SEO_IMG_DEFAULT} />
+      <meta name="twitter:site:domain" content="docs.cal.com" />
+      <meta name="twitter:url" content={DOCS_URL} />
+      <meta name="og:title" content="Cal.com Documentation" />
+      <meta name="og:image" content={SEO_IMG_DEFAULT} />
+      <meta name="apple-mobile-web-app-title" content="Cal.com Docs" />
```
Comment (dmkav) [inline apps/docs/joinTheme.copy.js:28]: cal.com?

File: apps/docs/joinTheme.copy.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      />
+      <meta
+        name="og:description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:image" content={SEO_IMG_DEFAULT} />
+      <meta name="twitter:site:domain" content="docs.cal.com" />
+      <meta name="twitter:url" content={DOCS_URL} />
+      <meta name="og:title" content="Cal.com Documentation" />
```
Comment (dmkav) [inline apps/docs/joinTheme.copy.js:26]: cal com?

File: apps/docs/joinTheme.copy.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        name="description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta
+        name="og:description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:image" content={SEO_IMG_DEFAULT} />
+      <meta name="twitter:site:domain" content="docs.cal.com" />
```
Comment (dmkav) [inline apps/docs/joinTheme.copy.js:24]: cal.com?

File: apps/docs/joinTheme.copy.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import { SEO_IMG_DEFAULT, DOCS_URL } from "@calcom/lib/constants";
+
+const themeConfig = {
+  github: "https://github.com/calcom/cal.com",
```
Comment (dmkav) [inline apps/docs/joinTheme.copy.js:4]: cal com?

File: apps/docs/joinTheme.config.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      <meta httpEquiv="Content-Language" content="en" />
+      <meta
+        name="description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta
+        name="og:description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta name="og:title" content="Join.com Documentation" />
```
Comment (dmkav) [inline apps/docs/joinTheme.config.js:20]: this is not Documnetation

it address to link you removed. Just set Join Calendar

File: apps/docs/joinTheme.config.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+      <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#000000" />
+      <meta name="msapplication-TileColor" content="#ff0000" />
+      <meta name="theme-color" content="#ffffff" />
+    </>
+  ),
+  search: true,
+  prevLinks: true,
+  nextLinks: true,
+  footer: true,
+  footerEditLink: "Edit this page on GitHub",
```
Comment (dmkav) [inline apps/docs/joinTheme.config.js:37]: we don;t have edit page

File: apps/docs/joinTheme.config.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+      <meta
+        name="description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta
+        name="og:description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta name="og:title" content="Join Calendar" />
+      <meta name="og:image" content={SEO_IMG_DEFAULT} />
```
Comment (dmkav) [inline apps/docs/joinTheme.config.js:21]: common man, can you check line by line to remove everything that related to cal.com?  Or you are playing around with GPT?))

Comment (MichalBaran94) [reply apps/docs/joinTheme.config.js:21]: I think that only thing that left is this line `<meta name="og:image" content={SEO_IMG_DEFAULT} />` 
but this var `SEO_IMG_DEFAULT` is pointing to `export const SEO_IMG_DEFAULT = `${WEBSITE_URL}/og-image.png`;` and it points to 
`export const WEBSITE_URL = process.env.NEXT_PUBLIC_WEBSITE_URL || "https://join.com";` so we can configure it throught our env variable and upload appropriate file to the server 🤔

Comment (dmkav) [reply apps/docs/joinTheme.config.js:21]: but we don't have access to logo. In this case if you will copy link to our website, picture will be broken. Better to remove it

**Review by dmkav — APPROVED** [review]

**Pull request merged**